src/app/services/restaurant.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { IRestaurant } from '../models/restaurant.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RestaurantService {
+  private baseUrl: string = 'http://localhost:3000/restaurants';
```

Comment:
Do you have an env file? put the base link there and use it something like this:

environment.apiUrl + '/restaurants'